install_config/storage_examples/gluster_backed_registry.adoc

```diff
@@ -0,0 +1,137 @@
+= Backing Docker Registry with GlusterFS Storage
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap:
+
+toc::[]
+
+== Overview
+
+This guide will cover the how to attach a GlusterFS persistent volume to the Docker Registry.
+It is assumed that the docker registry service has already been started and the gluster volume has been created.
+
+
+=== Prerequisites
+* The link:../install/docker_registry.html#deploy-registry[docker-registry] has been deployed *without*
+configuring storage.
+* A gluster volume exists and *glusterfs-fuse* is installed on all hosts.
+* Definitions written for GlusterFS link:../persistent_storage/persistent_storage_glusterfs.html#creating-gluster-endpoints[endpoints and service],
+link:../persistent_storage/persistent_storage_glusterfs.html#gfs-creating-persistent-volume[persistent volume (*pv*)],
+and
+link:../architecture/additional_concepts/storage.html#persistent-volume-claims[persistent volume claim (*pvc*)].
+** For this guide, these will be:
+*** `gluster-endpoints-service.yaml`
+*** `gluster-endpoints.yaml`
+*** `gluster-pv.yaml`
+*** `gluster-pvc.yaml`
+* A user with the
+link:../admin_guide/manage_authorization_policy.html#managing-role-bindings[*cluster-admin*] role binding.
+** For this guide, that user is `admin`
+
+NOTE:  All `$ oc` commands are executed on the master node as the `admin` user
+
+==
+
+[[create-gfs-pvc]]
+=== Create the Gluster Persistent Volume
+
+First, make the Gluster volume available to the registry.
+
+----
+$ oc create -f gluster-endpoints-service.yaml
+$ oc create -f gluster-endpoints.yaml
+$ oc create -f gluster-pv.yaml
+$ oc create -f gluster-pvc.yaml
+----
+
+[[attach-]]
+=== Attach the PVC to the Docker Registry
+
+Before moving forward, ensure that the docker-registry service is running.
+
+----
+$ oc get svc
+NAME              CLUSTER_IP       EXTERNAL_IP   PORT(S)                 SELECTOR                  AGE
+docker-registry   172.30.167.194   <none>        5000/TCP                docker-registry=default   18m
+----
+
+Then attach the *pvc*.
+
+----
+$ oc volume deploymentconfigs/docker-registry --add --name=v1 -t pvc \
+     --claim-name=<pvc_name> --overwrite
```

Comment:
do you want to use the real pvc name here? I think I read somewhere that there's a convention to use the real names from the sample files, but not positive...